module/VuFind/src/VuFind/Autocomplete/Solr.php

```diff
@@ -82,6 +82,13 @@ class Solr implements AutocompleteInterface
      */
     protected $sortField;
 
+    /**
+     * Max number of search result rows
+     *
+     * @var int
+     */
+    protected $limit;
```

Comment:
We should initialize this to be sure its behavior is predictable; making it nullable also changes the type:

```suggestion
    /**
     * Max number of search result rows
     *
     * @var ?int
     */
    protected ?int $limit = null;
```